pkg/ovs/openflow/ofctrl_action.go

```diff
@@ -330,6 +331,16 @@ func (a *ofFlowAction) SendToController(reason uint8) FlowBuilder {
 	return a.builder
 }
 
+func (a *ofFlowAction) Meter(meterId uint32) FlowBuilder {
+	// TODO: remove OS detection after windows OVS support OF meter.
+	// Skip apply OF meter for windows ovs for now.
+	if runtime.GOOS == "windows" {
```

Comment:
we have `runtime.IsWindowsPlatform()` for this

Reply:
Changed.